lib/customer.rb

```diff
@@ -0,0 +1,37 @@
+require "csv"
+
+class Customer
+  attr_reader :id
+  attr_accessor :email, :address
```

Comment:
email and address do not need to be accessors.  In general default to only give reader access unless you have a reason to give users direct access to your instance variables.